cfgov/v1/tests/wagtail/test_organisms.py

```diff
@@ -0,0 +1,224 @@
+from django.test import TestCase
+from django.test import Client
+
+
+from scripts import _atomic_helpers as atomic
+from v1.models.landing_page import LandingPage
+from v1.models.sublanding_page import SublandingPage
+from v1.models.learn_page import LearnPage
+from v1.models.browse_page import BrowsePage
+from wagtail.wagtailcore.blocks import StreamValue
+from helpers import publish_page
+from v1.models.snippets import Contact
+
+c = Client()
+
+'''
+TODO: Create tests for the following organisms:
+          - FilterableListControls
+          - Sidebar Breakout
+          - PostPreviewSnapshot
+          - RelatedPosts
+'''
+
+
+class OrganismsTestCase(TestCase):
+	def get_contact(self):
+		contact = Contact(heading='Test User')
+		contact.contact_info = StreamValue(
+			contact.contact_info.stream_block,
+			[
+				atomic.contact_email,
+				atomic.contact_phone,
+				atomic.contact_address
+			], 
+			True
+		)
+		contact.save()
+		return contact 
+
+	def test_well(self):
+		"""Well content correctly displays on a Landing Page"""
+		landing_page = LandingPage(
```

Comment:
readability and abstraction aren't mutually exclusive.  abstraction helps to make the code more readable.

Robert Martin says, "Duplicate code always represents a missing abstraction"

http://www.informit.com/articles/article.aspx?p=1313447

This is also fairly intuitive to understand.  Say, the slug-name changes for the landing page, or the url we want to load the response from.  As written, one would have to make that fairly simple change in a lot of different places, increasing the time and cost it takes to maintain.